docs/source/03_tutorial/05_visualise_pipeline.md

```diff
@@ -17,7 +17,7 @@ kedro viz
 ```
 
 This command will run a server on http://127.0.0.1:4141 that will open up your visualisation on a browser. You should
- be able to see the following:
+be able to see the following:
```

Comment:
can we actually just move it on the same line? We don't break up long lines in our .md files, we just use wrap-around in IDE.